change/end-to-end-test-app-node-2020-10-23-18-12-29-node-e2e-workflow.json

```diff
@@ -0,0 +1,8 @@
+{
+  "type": "none",
```

Comment:
This change file should not be needed. Double check your beachballrc file

Reply:
Was missing "private": true in package.json, fixed!